azure-sdk-testutils/devtools_testutils/mgmt_settings_fake.py

```diff
@@ -20,8 +20,7 @@
 CLIENT_OID = '00000000-0000-0000-0000-000000000000'
 # Cognitive Services tests
 CS_SUBSCRIPTION_KEY = '0000000000000000000000000000'
-# Event Grid key
-EVENT_GRID_KEY = '0000000000000000000000000000'
+
```

Comment:
This is why the EventGrid tests are failing. You should not change this file at all in your PR